janusgraph-server/src/main/java/org/janusgraph/graphdb/grpc/schema/SchemaManagerImpl.java

```diff
@@ -56,4 +57,36 @@ public void getVertexLabelByName(
         responseObserver.onNext(GetVertexLabelByNameResponse.newBuilder().setVertexLabel(vertexLabel).build());
         responseObserver.onCompleted();
     }
+
+    @Override
+    public void getEdgeLabelByName(
+        GetEdgeLabelByNameRequest request,
+        StreamObserver<GetEdgeLabelByNameResponse> responseObserver
+    ) {
+        if (request == null) {
+            responseObserver.onError(Status.INTERNAL
+                .withDescription("request is required").asRuntimeException());
+            return;
+        }
+        if (!request.hasContext()) {
+            responseObserver.onError(Status.INVALID_ARGUMENT
+                .withDescription("context is required").asException());
+            return;
+        }
+        final String edgeLabelName = request.getName();
+        if (edgeLabelName.isEmpty()) {
+            responseObserver.onError(Status.INVALID_ARGUMENT
+                .withDescription("name is required").asException());
+            return;
+        }
+        SchemaManagerProvider provider = contextHandler.getSchemaManagerProviderByContext(request.getContext());
+        EdgeLabel edgeLabel = provider.getEdgeLabelByName(edgeLabelName);
+        if (edgeLabel == null) {
+            responseObserver.onError(Status.NOT_FOUND.asException());
```

Comment:
(nitpick) I think we could provide a description here. NOT_FOUND can be due to the fact that graph does not exist, or edge label does not exist.